src/_addRemoveFunctions.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-alert */
+// eslint-disable-next-line import/no-named-default
+import checkboxesEvent, { list, saveLocalstorage } from './_checkboxFunctions.js';
+
+const textInput = document.getElementById('textInput');
+const addBtn = document.getElementById('addBtn');
+const li = document.getElementsByClassName('dropzone');
+const removeBtn = document.getElementsByClassName('removeBtn');
+class Task {
+  constructor(description, completed = false, index = (list.length)) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  addTask() {
+    list.push(this);
+  }
+}
+
+export default function editItem() {
+  for (let i = 0; i < li.length; i += 1) {
+    li[i].children[1].addEventListener('dblclick', () => {
+      const newInput = document.createElement('input');
+      const placeholder = li[i].children[1];
+      li[i].removeChild(li[i].children[1]);
+      newInput.placeholder = placeholder.innerHTML;
+      li[i].insertBefore(newInput, li[i].children[1]);
+      li[i].children[2].classList.add('d-none');
+      li[i].children[3].classList.remove('d-none');
+
+      newInput.addEventListener('keypress', (event) => {
+        if (event.key === 'Enter') {
+          if (newInput.value.trim() === '') {
+            newInput.classList.add('border', 'border-danger');
+          } else {
+            list[li[i].children[0].id].description = newInput.value;
+            li[i].removeChild(li[i].children[1]);
+            placeholder.innerHTML = `${newInput.value}`;
+            li[i].insertBefore(placeholder, li[i].children[1]);
+            li[i].children[3].classList.add('d-none');
+            li[i].children[2].classList.remove('d-none');
+            saveLocalstorage();
+          }
+        }
+      });
+    });
+  }
+}
+
+export function removeBtnEvents() {
+  for (let i = 0; i < removeBtn.length; i += 1) {
+    removeBtn[i].addEventListener('click', () => {
+      if (window.confirm('Are you sure to delete this task?')) {
```

Comment:
Change the confirm message to `'Are you sure?'` instead of `'Are you sure to delete this task?'`

Reply:
Fixed!